class TournamentNotFoundError(Exception):
    pass


class UserAlreadyExistsError(Exception):
    pass


class TournamentFullError(Exception):
    pass
